data/json/recipes/other/materials.json

```diff
@@ -126,6 +126,82 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "copper", 50 ] ] ]
   },
+  {
+    "result": "copper",
+    "id_suffix": "medium_batch",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "15 m",
+    "batch_time_factors": [ 90, 1 ],
+    "autolearn": true,
+    "charges": 50,
+    "using": [ [ "forging_standard", 1 ] ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
+    "components": [ [ [ "scrap_copper", 1 ], [ "copper_ear", 2 ], [ "cable", 50 ], [ "coin_dime", 25 ], [ "coin_nickel", 10 ] ] ]
```

Comment:
Cable is plastic jacketed, I imagine you need to remove the jacketing to melt it down.
Nickels and dimes are 1/4 and 1/12 nickel respectively - I don't think you can just melt those down into copper that simply either (but I could be wrong!).

Reply:
> Cable is plastic jacketed, I imagine you need to remove the jacketing to melt it down. Nickels and dimes are 1/4 and 1/12 nickel respectively - I don't think you can just melt those down into copper that simply either (but I could be wrong!).

For the temperatures involved in forging (1085c minimum, expected overage) the plastic coating can be expected to completely pyrolyse and decompose in the forge as slag. 

Nickel is a bit of a tougher question - the game doesn't have nickel to handle, or else a byproduct would be expected of both the deconstruction and smelting process. It's currently not accounted for in the deconstruction either. We don't appear to handle this question in any forging-related applications where you'd see mixed products. It might be reasonable to spin off coin recipes with a washing soda, as this is apparently a common and fairly simple method to separate the two, with the nickel not being yielded as it's assumed the player is discarding it in the same way they currently discard all the other unwanted side products in the recipe system (the majority produce a several).

Reply:
That sounds like a good solution to me - though the penny should just be ineligible, (2.5% copper), and the dollar coin is a more complex alloy, so that probably would not work for it.

Reply:
I've removed the offending coins and added washing soda to the recipies.